sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Microsoft.Azure.Management.HealthcareApis.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HealthcareApis</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure Healthcare Apis.</Description>
+    <AssemblyTitle>Microsoft Azure Healthcare Apis Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.HealthcareApis</AssemblyName>
+    <Version>1.0.5-preview</Version>
```

Comment:
Suggest to use major version 0 for brand new preview feature, e.g. 0.8.0-preview